fvm/evm/stdlib/contract.cdc

```diff
@@ -46,8 +46,8 @@ contract EVM {
         index: UInt16,
```

Comment:
It's great to see making the type less "stringly typed" 👍 

Can't comment on the right line, but maybe the hash an also be changed in a similar way?

Reply:
Going to create a follow-up PR for it. good idea.

Reply:
Nice! Have already some commits here if you want to grab them: https://github.com/onflow/flow-go/compare/ramtin/evm-avoid-hex-encoding-for-event-bytes...bastian/evm-improve-types